### ISSUE TITLE
Update `pastel` to `~> 0.8.0`

### ISSUE THREAD

**Issue by apainintheneck** [piotrmurach/tty-prompt#205]
### Describe the problem

I tried adding both `tty-markdown` and `tty-prompt` to the same project but `tty-prompt` depends on an older version of `pastel`.

```
Because tty-markdown >= 0.7.0 depends on pastel ~> 0.8
  and tty-prompt >= 0.10.0, < 0.22.0 depends on pastel ~> 0.7.0,
  tty-markdown >= 0.7.0 is incompatible with tty-prompt >= 0.10.0, < 0.22.0.
And because every version of gemview depends on tty-markdown ~> 0.7.2,
  tty-prompt >= 0.10.0, < 0.22.0 is incompatible with gemview >= 0.
```

### Steps to reproduce the problem
```rb
  spec.add_dependency "tty-prompt", "~> 0.21.0"
  spec.add_dependency "tty-markdown", "~> 0.7.2"
```

Add the following to the gemspec for a new gem and run `bundle install`.

### Actual behaviour
There is a conflict so that the two latest versions of the `tty-markdown` and `tty-prompt` gems can't be installed at the same time.

### Expected behaviour
It should be possible to install both gems in the same project.

### Describe your environment

* OS version: MacOS Catalina 10.15.7
* Ruby version: Ruby 3.2.2
* TTY::Prompt version: 0.21.0

**Comment by apainintheneck** [comment]
I didn't realize that it's already been upgraded in this repo but that hasn't been included in a release yet. When you've got time, could you cut a new release? It seems like the last published release came out in 2021.

**Comment by piotrmurach** [comment]
Hi Kevin,

Thanks for using `tty-prompt`.

There is no need to release anything. The error message and reproduction steps already tell you the problem. You're using version `0.21.0`. As you rightly noticed, a version released in 2021, specifically `0.23.1`, matches the `pastel` version constraint. I'm not sure why you haven't upgraded to it and keep on using an older version. I always do due diligence when releasing new libraries to ensure they're compatible.

**Comment by apainintheneck** [comment]
You're 100% right. I think I got confused because the top search result was `0.21.0` on rubygems.org and I just assumed that was the most recent one. I updated it and it's now working as expected. Sorry to bother you.
